scripts/rv_expression_association/get_vep_variants.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+
+import logging
+import hail as hl
+import numpy as np
+import pandas as pd
+from hail.methods import export_plink
+from cpg_utils.hail_batch import dataset_path, init_batch, output_path
+
+# object containing variants within a 50K window on either side of the VPREB3 gene
+MT = dataset_path('v0/VPREB3_50K_window.mt')
+VEP_HT = dataset_path('v0/VPREB3_50K_window_vep.ht')
+
+# use logging to print statements, display at info level
+logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.INFO)
+
+
+def main():
+    # read and densify object
+    init_batch()
+    mt = hl.read_matrix_table(MT)
+    mt = hl.experimental.densify(mt)
+
+    logging.info('Number of variants in window: {}'.format(mt.count()[0]))
+
+    # filter out low quality variants
+    mt = mt.filter_rows(hl.len(hl.or_else(mt.filters, hl.empty_set(hl.tstr))) == 0)
+
+    # consider biallelic variants only (no multi-allelic, no ref-only)
+    mt = mt.filter_rows(hl.len(mt.alleles) == 2)
+    mt = mt.filter_rows(hl.is_snp(mt.alleles[0], mt.alleles[1]))
```

Comment:
Consider combining these filter predicates, as they're not dependent on one another:

```suggestion
    mt = mt.filter_rows(
        # filter out low quality variants
        (hl.len(hl.or_else(mt.filters, hl.empty_set(hl.tstr))) == 0) &
        # consider biallelic variants only (no multi-allelic, no ref-only)
        (hl.len(mt.alleles) == 2) &
        hl.is_snp(mt.alleles[0], mt.alleles[1]))
```